string blocker_id
string blocked_id
time stamp